include/verilated_random.h

```diff
@@ -121,74 +131,29 @@ class VlRandomQueueVar final : public VlRandomVar {
             const auto it = m_arrVarsRefp->find(indexed_name);
             if (it != m_arrVarsRefp->end()) {
                 const std::vector<size_t>& indices = it->second->m_indices;
-                emitSelect(s, indices);
+                const std::vector<size_t>& idx_widths = it->second->m_idx_widths;
+                emitSelect(s, indices, idx_widths);
+            } else {
+                VL_FATAL_MT(__FILE__, __LINE__, "randomize",
+                            "Error: indexed_name not found in m_arr_vars");
```

Comment:
```suggestion
                            "indexed_name not found in m_arr_vars");
```

Similar other places (FATAL code prints %Error itself).